mne/evoked.py

```diff
@@ -358,7 +358,7 @@ def ch_names(self):
     def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
              xlim='tight', proj=False, hline=None, units=None, scalings=None,
              titles=None, axes=None, gfp=False, window_title=None,
-             spatial_colors=False, zorder='unsorted', selectable=True,
+             spatial_colors='auto', zorder='unsorted', selectable=True,
              noise_cov=None, time_unit='s', sphere=None, *, highlight=None,
              verbose=None):
         return plot_evoked(
```

Comment:
I suspect you need to update some docstrings

thx @HuseyinOrkun

Reply:
thx @agramfort. I checked but couldn't find which docstrings I still need to update. Could you give me some pointers?